wgsl/wgsl.recursive.bs.include

```diff
@@ -376,12 +372,6 @@
  | [=syntax/star=]
 </div>
 
-<div class='syntax' noexport='true'>
-  <dfn for='recursive descent syntax'>optionally_typed_ident</dfn>:
```

Comment:
I don't think this should have been inlined?

Reply:
Added it to the exclusion list of things to be inlined.  Reads better that way.